File: tcp_udp_experiment/vps_checker/notificate.py
import requests
import os 
from datetime import datetime

WEBHOOK_URL = os.environ["SLACK_WEBHOOK_URL"]

#message = "hello world from PYTHON"
#date = "every day 17:00" 日付は cron で指定して送りましょう

message = {"text": f"\n{datetime.now().strftime("%Y-%m-%d %H:%M:%S")} \nhello world from PYTHON !"}
status = requests.post(WEBHOOK_URL, json=message) 
print(status.status_code, status.text)

log_file_name = "post_check.log"
with open(log_file_name, "a") as f:
    f.write(f"\n{datetime.now().strftime("%Y-%m-%d %H:%M:%S")}: {status.status_code} {status.text}")

File: tcp_udp_experiment/vps_checker/notificate.js
const WEBHOOK_URL=process.env.SLACK_WEBHOOK_URL

async function send(){
    let response = await fetch(WEBHOOK_URL, {
                        method: "POST",
                        headers: {"Content-Type": "application/json"},
                        body: JSON.stringify({"text": "hello from javascript"})
                    })

    const status = response.status
    const text = await response.text()

    console.log(status, text)

    const now = new Date().toISOString();
    require("fs").appendFileSync("post_check.log", `\n${now}: ${status} ${text}`)
}

send()